models/repository.go

```diff
@@ -0,0 +1,22 @@
+package models
+
+import (
+	"github.com/almighty/almighty-core/app"
+	"github.com/almighty/almighty-core/criteria"
+	"golang.org/x/net/context"
+)
+
+// WorkItemRepository encapsulates storage & retrieval of work items
+type WorkItemRepository interface {
+	Load(ctx context.Context, ID string) (*app.WorkItem, error)
+	Save(ctx context.Context, wi app.WorkItem) (*app.WorkItem, error)
+	Delete(ctx context.Context, ID string) error
+	Create(ctx context.Context, typeID string, fields map[string]interface{}, creator string) (*app.WorkItem, error)
+	List(ctx context.Context, criteria criteria.Expression, start *int, length *int) ([]*app.WorkItem, error)
+}
+
+type WorkItemTypeRepository interface {
```

Comment:
This interface already moved to `github.com/almighty/almighty-core/application` package, right?